package.json

```diff
@@ -110,10 +110,12 @@
     "json-loader": "^0.5.4",
     "jsx-to-string": "^0.2.13",
     "less-vars-to-js": "^1.1.1",
-    "lodash.debounce": "^4.0.6",
-    "lodash.flatten": "^4.4.0",
-    "lodash.omit": "^4.5.0",
-    "lodash.range": "^3.2.0",
+    "lodash.debounce": "4.0.6",
+    "lodash.flatten": "4.4.0",
+    "lodash.get": "4.4.2",
```

Comment:
The issue is that dev dependencies won't get installed by consumers, so they won't have these packages available.